test/fixtures/categories.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
Please do not commit test files. Delete them right away when they are created. Because we are not writing test cases in the project.